client/src/app/pages/package-list/package-table.tsx

```diff
@@ -55,72 +67,124 @@ export const PackageTable: React.FC = () => {
+                        <Td
+                          width={15}
+                          modifier="breakWord"
+                          {...getTdProps({ columnKey: "name" })}
+                        >
+                          <NavLink
+                            to={generatePath(Paths.packageDetails, {
+                              packageId: item.uuid,
+                            })}
+                          >
+                            {item.decomposedPurl
+                              ? item.decomposedPurl?.name
+                              : item.purl}
+                          </NavLink>
+                        </Td>
+                        <Td
+                          width={15}
+                          modifier="truncate"
+                          {...getTdProps({ columnKey: "namespace" })}
+                        >
+                          {item.decomposedPurl?.namespace}
+                        </Td>
+                        <Td
+                          width={10}
+                          modifier="truncate"
+                          {...getTdProps({ columnKey: "version" })}
+                        >
+                          {item.decomposedPurl?.version}
+                        </Td>
+                        <Td
+                          width={10}
+                          modifier="truncate"
+                          {...getTdProps({ columnKey: "type" })}
+                        >
+                          {item.decomposedPurl?.type}
+                        </Td>
+                        <Td
+                          width={10}
+                          modifier="truncate"
+                          {...getTdProps({
+                            columnKey: "licenses",
+                            isCompoundExpandToggle: true,
+                            item,
+                            rowIndex,
+                          })}
+                        >
+                          {pkg?.licenses?.length ?? 0}{" "}
+                          {(pkg?.licenses?.length ?? 0) > 1
+                            ? "Licenses"
+                            : "License"}
+                        </Td>
+                        <Td
+                          width={10}
+                          modifier="truncate"
+                          {...getTdProps({ columnKey: "path" })}
+                        >
+                          {item.decomposedPurl?.path}
+                        </Td>
+                        <Td
+                          width={20}
+                          {...getTdProps({ columnKey: "qualifiers" })}
+                        >
+                          {item.decomposedPurl?.qualifiers && (
+                            <PackageQualifiers
+                              value={item.decomposedPurl?.qualifiers}
+                            />
+                          )}
+                        </Td>
+                        <Td
+                          width={20}
```

Comment:
Not necessarily this line but make sure the total sum of `width` is 100. (taking all columns).,.. right now I think it exceeds 100.